README.md

```diff
@@ -15,6 +16,7 @@ Currently supported platforms are Windows and Linux, x86_64 only.
 * `g++` compiler
 * `pkg-config` tool
 * `cmake` tool
+* `python-config` symlinked to the `python3-config`
```

Comment:
Where does this come from? In my Ubuntu 16.04 VM it is enough to specify just the Python, CMake handles the rest.

Reply:
Ah, that is about building prerequisites... Still worth noting that it is needed if you want to build against python3 (not needed for python2).